metricbeat/mb/testing/data/data_test.go

```diff
@@ -210,8 +246,14 @@ func checkDocumented(t *testing.T, data []common.MapStr) {
 
 	for _, d := range data {
 		flat := d.Flatten()
+	keys:
 		for k := range flat {
 			if _, ok := keys[k]; !ok {
+				for _, omitField := range omitFields {
```

Comment:
Could you make this a function instead so we would have here a if statement instead and can directly use `continue` without needing `keys`?

Reply:
Done!